doc/source/cd/methods/cvmdrift.ipynb

```diff
@@ -94,16 +94,33 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
```

Comment:
Out of curiosity what is this metadata? I assume it doesn't matter if it's here in the published versions?

Reply:
This is accidental. Think it was added as I opened the file in PyCharm. Will remove.